src/seraphis_main/enote_record_types.h

```diff
@@ -129,30 +129,34 @@ struct LegacyEnoteRecord final
 
 ////
 // SpBasicEnoteRecordV1
-// - a seraphis enote that has passed the view-tag check using a jamtis find-received key
+// - a seraphis enote that resides in a transaction with an enote that passes the primary view tag check
 ///
-struct SpBasicEnoteRecordV1 final
+struct SpBasicEnoteRecordV1
 {
     /// original enote
     SpEnoteVariant enote;
     /// the enote's ephemeral pubkey
     crypto::x25519_pubkey enote_ephemeral_pubkey;
+    // the transaction primary view tag size
+    std::uint8_t num_primary_view_tag_bits;
     /// context of the tx input(s) associated with this enote
     rct::key input_context;
-    /// t'_addr: nominal address tag (only useful for jamtis non-selfsend enote types)
-    jamtis::address_tag_t nominal_address_tag;
+    /// did this enote not pass the check?
+    bool is_auxiliary;
```

Comment:
```suggestion
    /// indicates if this enote passed the exclusive-enote view tag check
    /// - If it did not pass, then it *might* be an auxiliary enote.
    bool passed_exclusive_check;
```
The `is_auxiliary` you have is ambiguous.